### PR TITLE
feat: Deprecate SheetBottom component 

### PR DESCRIPTION
**Pull request by tommasini**
## **Description**
Deprecating SheetBottom component, for developers use the [BottomSheet](https://github.com/MetaMask/metamask-mobile/blob/2a1a46a23929d36553c8499b535edef161e200c5/app/component-library/components/BottomSheets/BottomSheet/BottomSheet.tsx#L33) instead.

## **Related issues**

Fixes: #

## **Manual testing steps**

1. Go to this page...
2.
3.

## **Pre-merge author checklist**

- [ ] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've clearly explained what problem this PR is solving and how it is solved.
- [ ] I've linked related issues
- [ ] I've included manual testing steps
- [ ] I've included screenshots/recordings if applicable
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.
- [ ] I’ve properly set the pull request status:
  - [ ] In case it's not yet "ready for review", I've set it to "draft".
  - [ ] In case it's "ready for review", I've changed it from "draft" to "non-draft".

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

File: app/component-library/components/Sheet/SheetBottom/SheetBottom.tsx [hunk 1]
```diff
@@ -57,6 +57,13 @@ import {
   SheetBottomRef,
 } from './SheetBottom.types';
 
+/**
+ * @deprecated The `<SheetBottom>` component has been deprecated in favor of the new `<BottomSheet>` component from the component-library.
+ * Please update your code to use the new `<BottomSheet>` component instead, which can be found at app/component-library/components/BottomSheets/BottomSheet/BottomSheet.tsx.
+ * You can find documentation for the new BottomSheet component in the README:
+ * {@link https://github.com/MetaMask/metamask-mobile/tree/main/app/component-library/components/BottomSheets/BottomSheet/README.md}
+ * If you would like to help with the replacement of the old SheetBottom component, please create a new GitHub issue to address a Pull request to it.
```
Comment (georgewrmarshall) [inline app/component-library/components/Sheet/SheetBottom/SheetBottom.tsx:65]: I've created a good first issue that contributors can reference if they'd like to replace the old SheetBottom with BottomSheet

```suggestion
 * If you would like to help with the replacement of the old `OLD_COMPONENT` component, please submit a pull request against this GitHub issue:
 * {@link https://github.com/MetaMask/metamask-mobile/issues/7658}
```

**Review by georgewrmarshall — COMMENTED** [review]
Excellent work! Made one suggestion
- Pulled branch and checked deprecated formatting works ✅ 

Screenshot 2023-10-31 at 6 27 19 PM

File: app/component-library/components/Sheet/SheetBottom/SheetBottom.tsx (hunk 1 above)
Comment (tommasini) [reply app/component-library/components/Sheet/SheetBottom/SheetBottom.tsx:65]: Thank you so much for that @georgewrmarshall <3

**Review by Cal-L — APPROVED** [review]
LGTM

**Review by georgewrmarshall — APPROVED** [review]

**Pull request merged**